client/src/components/forms/CommentEditor.vue

```diff
@@ -56,25 +56,35 @@
       </q-list>
     </div>
     <q-card-actions class="q-mt-md q-pa-none" align="between">
-      <q-btn data-ref="submit" color="primary" @click="submitHandler()">{{
+      <q-btn data-cy="submit" color="primary" @click="submitHandler()">{{
         $t("guiElements.form.submit")
       }}</q-btn>
```

Comment:
The whole form needs to be wrapped in a q-form with a @submit handler on the q-form.  Then this button becomes type submit.

Reply:
This has been refactored as suggested.